template/html/integrations.html

```diff
@@ -108,6 +108,21 @@ <h3>Nunux Keeper</h3>
         <input type="text" name="nunux_keeper_api_key" id="form-nunux-keeper-api-key" value="{{ .form.NunuxKeeperAPIKey }}">
     </div>
 
+    <h3>Pocket</h3>
+    <div class="form-section">
+        <label>
+            <input type="checkbox" name="pocket_enabled" value="1" {{ if .form.PocketEnabled }}checked{{ end }}> {{ t "Save articles to Pocket" }}
+        </label>
+
+        <label for="form-pocket-access-token">{{ t "Pocket Access Token" }}</label>
+        <input type="password" name="pocket_access_token" id="form-pocket-access-token" value="{{ .form.PocketAccessToken }}">
+        (<a href="https://www.jamesfmackenzie.com/getting-started-with-the-pocket-developer-api/">{{ t "How To" }}</a>)
```

Comment:
If you don't mind, I would prefer to have the documentation on the official website like the other integration: https://docs.miniflux.net/en/latest/integrations.html

Reply:
:+1: